tools/cn_metrics/cn_metrics.c

```diff
@@ -361,19 +362,19 @@ print_ids(
 
 const char *hdrv[] = { "H", "Loc", "Dgen", "Keys", "Tombs", "Ptombs", "AvgKlen",
                        "AvgVlen", "HbAlen", "KbAlen", "VbAlen", "HbWlen%",
-                       "KbWlen%", "VbWlen%", "VbUlen%", "Comps", "Kbs", "Vbs" };
+                       "KbWlen%", "VbWlen%", "VbUlen%", "Comps", "Vgrps", "Kbs", "Vbs" };
 
 #define FMT_HDR                        \
```

Comment:
This stuff was so hard to get right in the hblock PR. Will never touch it ever again.